client/state/jetpack/connection/test/actions.js

```diff
@@ -89,4 +94,73 @@ describe( 'actions', () => {
 			} );
 		} );
 	} );
+
+	describe( '#disconnect()', () => {
+		const status = {
+			success: true,
+			message: 'Site Name has been disconnected.'
+		};
+
+		describe( 'success', () => {
+			useNock( ( nock ) => {
+				nock( 'https://public-api.wordpress.com:443' )
+					.persist()
+					.post( '/rest/v1.1/jetpack-blogs/' + siteId + '/mine/delete' )
+					.reply( 200, status, {
+						'Content-Type': 'application/json'
+					} );
+			} );
+
+			it( 'should dispatch a request connection status action when thunk triggered', () => {
```

Comment:
It probably makes sense to reword this as `disconnect request action` instead of `request connection status`.